shub/settings.py

```diff
@@ -0,0 +1,537 @@
+    # Should registries by default be private, with no option for public?
+    "PRIVATE_ONLY": False,
+    # Should the default for a new registry be private or public?
+    "DEFAULT_PRIVATE": False,
+    # Disable all pushes of containers, recipes, etc. Also for Google Cloud Build
+    "DISABLE_BUILDING": False,
+    # A global setting to disable all webhooks / interaction with Github
+    "DISABLE_GITHUB": False,
+    # prevent responses from being received from Google Cloud Build
+    "DISABLE_BUILD_RECEIVE": False,
+    # use SSL for minio
+    "MINIO_SSL": False,
+    "MINIO_MULTIPART_UPLOAD": True,
+    # Don't clean up images in Minio that are no longer referenced by sregistry
+    "DISABLE_MINIO_CLEANUP": False,
+    # Do you want to save complete response metadata per each pull?
+    # If you disable, we still keep track of collection pull counts, but not specific versions
+    "LOGGING_SAVE_RESPONSES": True,
+    # Given that someone goes over, are they blocked for the period?
+    "VIEW_RATE_LIMIT_BLOCK": True,
+}
+
+STRING_DEFAULTS = {
+    "API_VERSION": "v1",
+    "API_ANON_THROTTLE_RATE": "100/day",
+    "API_USER_THROTTLE_RATE": "1000/day",
+    "API_DEFAULT_SCHEMA_CLASS": "rest_framework.schemas.coreapi.AutoSchema",
+    "API_DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
+    "SOCIAL_AUTH_LOGIN_REDIRECT_URL": "http://127.0.0.1",
+    # On any admin or plugin login redirect to standard social-auth entry point for agreement to terms
+    "LOGIN_REDIRECT_URL": "/login",
+    # DOMAIN NAMES
+    ## IMPORTANT: if/when you switch to https, you need to change "DOMAIN_NAME"
+    # to have https, otherwise some functionality will not work (e.g., GitHub webhooks)
+    "DOMAIN_NAME": "http://127.0.0.1",
+    "DOMAIN_NAME_HTTP": "http://127.0.0.1",
+    # Get additional admins from the environment
+    "HELP_CONTACT_EMAIL": "vsoch@users.noreply.github.com",
+    "HELP_INSTITUTION_SITE": "https://srcc.stanford.edu",
+    "REGISTRY_NAME": "Tacosaurus Computing Center",
+    "REGISTRY_URI": "taco",
+    # "UA-XXXXXXXXX"
+    "GOOGLE_ANALYTICS": None,
+    "GOOGLE_BUILD_SINGULARITY_VERSION": "v3.3.0-slim",
+    "SREGISTRY_DATABASE_ENGINE": "django.db.backends.postgresql_psycopg2",
+    "SREGISTRY_DATABASE_NAME": "postgres",
+    "SREGISTRY_DATABASE_USER": "postgres",
+    "SREGISTRY_DATABASE_HOST": "db",
+    "SREGISTRY_DATABASE_PORT": "5432",
+    # Minio Storage
+    "MINIO_SERVER": "minio:9000",  # Internal to sregistry
+    "MINIO_EXTERNAL_SERVER": "127.0.0.1:9000",  # minio server for Singularity to interact with
+    "MINIO_BUCKET": "sregistry",
+    "MINIO_REGION": "us-east-1",
+    # The rate limit for each view, django-ratelimit, "50 per day per ipaddress)
+    "VIEW_RATE_LIMIT": "50/1d",
+    "DJANGO_LOG_LEVEL": "WARNING",
+    # An image url or one of the following: 'mm', 'identicon', 'monsterid', 'wavatar', 'retro'. Defaults to 'mm'
+    "GRAVATAR_DEFAULT_IMAGE": "retro",
```

Comment:
The comment says the default is `mm` but we are setting `retro` here.

Reply:
I'll clarify this - I think if it's unset it defaults to mm, and we override with retro.